backend/models/user-services.js

```diff
@@ -0,0 +1,149 @@
+const mongoose = require("mongoose");
```

Comment:
event-services and user-services technically don't follow the naming convention of camel case

Reply:
Do file names fall under the naming convention as well? I just copy pasted from previous assignments